include/fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp

```diff
@@ -79,15 +72,26 @@ struct sidb_simulation_parameters
      * µ- is the energy transition level (0/-) (unit: eV).
      */
     double mu;
```

Comment:
While we're at it and are renaming stuff, can we fix this inconsistency with `mu` and `mu_p` as well and simply rename them to `mu_minus` and `mu_plus`?

Reply:
sure! I will change it quickly.